examples/plot_dict_learning_resting_state.py

```diff
@@ -0,0 +1,84 @@
+"""
```

Comment:
I know I already said that: but some broad indicators of the conceptual difference between sparse PCA and dictionary learning would be helpful, along the lines of "Informally, think of sparse PCA (parsimony in voxels) as the transpose of dictionary learning (parsimony in components)."